puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/test_proxy_develop_pr_core.yaml

```diff
@@ -41,7 +41,7 @@
             - 4.2.0
             - 4.4.0
     execution-strategy:
-      combination-filter: !( (ruby ==~ /2\.[^0]*/ && puppet ==~ /3\.[0-4].*/) || (ruby ==~ /2\.[2-9].*/ && puppet ==~ /3.*/) || (ruby ==~ /2\.[3-9].*/ && puppet ==~ /4\.[0-3].*/) )
+      combination-filter: '!( (ruby ==~ /2\.[^0]*/ && puppet ==~ /3\.[0-4].*/) || (ruby ==~ /2\.[2-9].*/ && puppet ==~ /3.*/) || (ruby ==~ /2\.[3-9].*/ && puppet ==~ /4\.[0-3].*/) )'
```

Comment:
I copied this from all other YAML files that use a combination filter. Should we change them all?

Reply:
If the filter is short, I don't have a problem with it being on the same line. But a lot of the filters seem to be pretty huge, so I kinda like them on the next line.

Technically that's just personal preference because both are still interpreted as strings at the the end of the day.

Reply:
Eventually I would really like for us to have a unified style for our JJB yaml and groovy, but I'd consider that a lower priority at this point?

Reply:
Agreed. We seem to be moving to pipelines which are more explicit than combination filters so it might end up going away anyway.